release-engineering/platforms.md

```diff
@@ -0,0 +1,65 @@
+# Supported Platforms
+
+The Kubernetes release process produces [artifacts](./artifacts.md) for
+different architectures and operating systems. We consider the combination of
+architecture (`GOARCH`) and operating system (`GOOS`) as "platforms". Target of
+this document is to outline different categories of platforms as well as guiding
+through their graduation criteria.
+
+## Tiers
+
+Support for different platforms are organized into three tiers, whereas each
+comes with a different set of guarantees.
+
+### Tier 1
+
+Tier 1 platforms are considered as "guaranteed to work". To achieve this, they
+have to fulfil the following criteria:
+
+- **All** official binary releases are provided for the platform. This includes
+  container images as well as deb and rpm packages. Building the artifacts is
+  integrated in the release process.
+- Continuous Integration is set up to run tests for the platform. Necessary
+  tests are defined by SIG Release and usually correspond to the
+  [`blocking`](https://testgrid.k8s.io/sig-release-master-blocking) and
+  [`informing`](https://testgrid.k8s.io/sig-release-master-informing) testgrid
+  dashboards.
+- Documentation about the usage of artifacts for the platform is available.
+
+### Tier 2
+
+Tier 2 platforms are considered as "guaranteed to build". To achieve this, they
+have to fulfil the following criteria:
+
+- **Any** official binary release is provided for the platform. Building the
```

Comment:
This is kind of tricky because we currently release artifacts that are not well tested or tested at all, but in the future it would be great if we "released" only Tier 1 artifacts, and published Tier 2 separately. However, I think we have to stick with that is defined here for now given our current state.

Reply:
@hasheddan do we track this open flanks already?